adc:
  publish_topic: "/adc_data"
  acquisition_rate: 5
